zschema/compounds.py

```diff
@@ -33,9 +34,22 @@ def to_bigquery(self, name):
         retv["mode"] = "REPEATED"
         return retv
 
+    def docs_bq(self, parent_category=None):
+        retv = self.object_.docs_bq()
+        category = self.category if self.category else parent_category
```

Comment:
Should this be pulled out into a method?

Reply:
> Should this be pulled out into a method (e.g. `get_category()`)?

Good note, @justinbastress. While it might be an intuitive restructuring, it wouldn't gain us much since we still have to fallback to `parent_category` passed _into_ this method. Let's just leave the code as is (it could be made terser with `category = self.category or parent_category` , but meh).

Reply:
^ what Andrew said. I do like the more concise suggestion, implemented in 5136ebfc23eda82f8ef90cc4f1fd50bf950b0970